hledger-lib/Hledger/Data/Journal.hs

```diff
@@ -532,33 +534,14 @@ journalReverse j =
     ,jmarketprices     = reverse $ jmarketprices j
     }
 
+-- | Set clock time in journal
```

Comment:
"Set this journal's last read time, ie when its files were last read." ? Rename to journalSetLastReadTIme ?

Reply:
Sure -- I'll do another renaming/redoc-ing commit for this and the below, and add it to this PR.